tests/phpunit/tests/l10n/localeSwitcher.php

```diff
@@ -407,7 +408,7 @@ public function test_switch_reloads_plugin_translations_outside_wp_lang_dir() {
 
 		require_once DIR_TESTDATA . '/plugins/custom-internationalized-plugin/custom-internationalized-plugin.php';
 
-		$this->assertSame( WP_PLUGIN_DIR . '/custom-internationalized-plugin/languages/', $wp_textdomain_registry->get( 'custom-internationalized-plugin' ) );
+		$this->assertFalse( $wp_textdomain_registry->get( 'custom-internationalized-plugin' ) );
```

Comment:
When the initial locale does not have `.mo` file, the registered text domain returns `false`. However, this prevents the ability to switch locales during the same request, as witnessed in the rest of this test.